### PR TITLE
PMM-6899 cluster external connections

### PR DESCRIPTION
**Pull request by percona-csalguero**
[PMM-6899](https://jira.percona.com/browse/PMM-6899)

- [ ] Build: https://github.com/Percona-Lab/pmm-submodules/pull/1341

---
- [ ] Tests passed.
- [ ] Feature build pass.
- [ ] (Re)requested review.
- [ ] Fix conflicts with target branch.
- [ ] Update API dependency.
- [ ] Update jira ticket description if needed.
- [ ] Attach screenshots/console output to confirm new behavior to jira ticket, if applicable.

### CODE REVIEW

**Review by BupycHuk — APPROVED** [review]

File: services/management/dbaas/psmdb_cluster_service.go [hunk 1]
```diff
@@ -18,8 +18,8 @@ package dbaas
 
 import (
 	"context"
-	"fmt"
 
+	controllerv1beta1 "github.com/percona-platform/dbaas-api/gen/controller"
```
[the remark this answers is not in the document]
Comment (BupycHuk) [reply services/management/dbaas/psmdb_cluster_service.go:22]: @percona-csalguero ^

**Review by BupycHuk — CHANGES_REQUESTED** [review]

File: services/management/dbaas/xtradb_cluster_service.go [hunk 2]
```diff
@@ -19,8 +19,8 @@ package dbaas
 
 import (
 	"context"
-	"fmt"
 
+	controllerv1beta1 "github.com/percona-platform/dbaas-api/gen/controller"
```
[the remark this answers is not in the document]
Comment (BupycHuk) [reply services/management/dbaas/xtradb_cluster_service.go:23]: @percona-csalguero ^

**Comment by BupycHuk** [comment]
@percona-csalguero please fix merge conflicts

File: services/management/dbaas/xtradb_cluster_service.go [hunk 3]
```diff
@@ -114,16 +113,33 @@ func (s XtraDBClusterService) GetXtraDBCluster(ctx context.Context, req *dbaasv1
 	}
 
 	// TODO: implement on dbaas-controller side:
-	// 1. Get pxc host and status
+	// 1. Get --- done pxc host ---  and status
```
Comment (BupycHuk) [inline services/management/dbaas/xtradb_cluster_service.go:116]: let's remove done part from the comment

File: services/management/dbaas/psmdb_cluster_service.go [hunk 4]
```diff
@@ -102,21 +101,38 @@ func (s PSMDBClusterService) ListPSMDBClusters(ctx context.Context, req *dbaasv1
...
 func (s PSMDBClusterService) GetPSMDBCluster(ctx context.Context, req *dbaasv1beta1.GetPSMDBClusterRequest) (*dbaasv1beta1.GetPSMDBClusterResponse, error) {
-	_, err := models.FindKubernetesClusterByName(s.db.Querier, req.KubernetesClusterName)
+	kubernetesCluster, err := models.FindKubernetesClusterByName(s.db.Querier, req.KubernetesClusterName)
 	if err != nil {
 		return nil, err
 	}
 
 	// TODO: implement on dbaas-controller side:
-	// 1. Get psmdb host and status
+	// 1. Done in PMM-6899 --- Get psmdb host and status ---
```
Comment (BupycHuk) [inline services/management/dbaas/psmdb_cluster_service.go:110]: let's remove done part from the PR

Comment (BupycHuk) [reply services/management/dbaas/psmdb_cluster_service.go]: by done part I meant what have done by you. 
TODO should be up to date

Comment (percona-csalguero) [reply services/management/dbaas/psmdb_cluster_service.go]: Done

**Comment by AlekSi** [comment]
@percona-csalguero Merge conflicts

**Comment by percona-csalguero** [comment]
> @percona-csalguero Merge conflicts

Working on it and changes for CR

**Review by BupycHuk — APPROVED** [review]

**Review by askomorokhov — APPROVED** [review]

**Pull request merged**